### ISSUE TITLE
Process explorer: List is not stable, cannot be navigated with keyboard

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#120342]
Would help accessibility if the elements didn't jump around like this.

![recording (18)](https://user-images.githubusercontent.com/2193314/113295119-a11d1500-92ac-11eb-8c5f-5d427b1a5dd9.gif)

**Comment by joyceerhl** [comment]
Not reproducible for me, possibly addressed with adoption of the tree widget. Closing and assigning to current milestone for verification.
